modal/app.py

```diff
@@ -678,6 +678,12 @@ def wrapped(
                 is_generator = f.is_generator
                 batch_max_size = f.batch_max_size
                 batch_wait_ms = f.batch_wait_ms
+                if f.max_concurrent_inputs:  # Using @modal.concurrent()
```

Comment:
Should there be some kind of error if both @concurrent and allow_concurrent_inputs on the decorator is used? Maybe it would be sufficiently covered by the allow_concurrent_inputs deprecation?

Reply:
Yeah I think once we add that deprecation warning it'll be unambiguous that one replaces the other, if it's not clear now.